src/sc/recipe/staticresources/bobtemplate/package.json.bob

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
what happens to this file when running buildout again? what happens to the options if I add or remove something manually?

Reply:
Nothing happen.  This webpack [folder structure is created just if it don't exists](https://github.com/simplesconsultoria/sc.recipe.staticresources/pull/2/files#diff-7b7a2cd347045411db48e3650817a07dR85).